packages/web/package.json

```diff
@@ -15,11 +15,12 @@
   "main": "js/cjs/spirit-web.js",
   "module": "js/esm/spirit-web.js",
   "scripts": {
+    "prestart": "shx cp -r ../../node_modules/@lmc-eu/spirit-icons/dist/ dist/icons",
```

Comment:
Would it make any sense to have here the same as in the `prebuild` script?

Reply:
Sure, I created new script and I start it from both pre* scripts